### ISSUE TITLE
Perms issue in /home/tor2web/certs/

### ISSUE THREAD

**Issue by vjrj** [globaleaks/Tor2web#320]
I get these messages:
```
Tor2web Startup Failure: unexistent file (/home/tor2web/certs/tor2web-key.pem)
Tor2web Startup Failure: unexistent file (/home/tor2web/certs/tor2web-key.pem)
```
starting tor2web in a globaleaks site.

IMHO, this chmod:
```
tor2web.postinst:chmod 660 /home/tor2web/certs/ -R
```
should be:
```
tor2web.postinst:chmod 770 /home/tor2web/certs/ -R
```

**Comment by evilaliv3** [comment]
Nice catch @vjrj thanks!

**Comment by evilaliv3** [comment]
fixed, the patch will be shipped in next release (3.1.67)
